### PR TITLE
[SE-0313] Post-review revisions

### PR DESCRIPTION
**Pull request by DougGregor**
Proposal revisions based on the review:
* Remove closure isolation control.
* Prohibit multiple isolated parameters.

### CODE REVIEW

File: proposals/0313-actor-isolation-control.md [hunk 1]
```diff
@@ -88,6 +88,20 @@ This makes instance methods on actor types less special, because now they are ex
...
+func f(a: isolated BankAccount, b: isolated BankAccount) {  // error: multiple isolated parameters in function `f(a:b:)`.
+  // ...
+}
+
+extension BankAccount {
+  func quickTransfer(amount: Double, to other: isolated BankAccount) {  // error: multiple isolated parameters in function 'quickTransfer(amount:to:)'
+    // ...
+  }
+}
+```
```
Comment (ktoso) [inline proposals/0313-actor-isolation-control.md:103]: 👍

File: proposals/0313-actor-isolation-control.md [hunk 2]
```diff
@@ -333,7 +305,7 @@ At a high level, isolated parameters and isolated conformances are similar to pa
       func g() {
         f(self) // T = @sync MyActor
 
-        Task.runDetached {
+        asyncDetached {
```
Comment (ktoso) [inline proposals/0313-actor-isolation-control.md:308]: 👍

**Review by ktoso — APPROVED** [review]
LGTM :)

**Comment by lattner** [comment]
LGTM

**Review by tkremenek — APPROVED** [review]

**Pull request merged**